### PR TITLE
Adds new reconciliation guidance and combines existing guidance

### PR DESCRIPTION
**Pull request by NathanD-GDS**
### Context
In early 2023, Pay's content designers drafted some reconciliation guidance to live in the tech docs. This ticket is the publication of that draft, as well as some additions related to the API.

### Changes proposed in this pull request
- renames the 'reporting on a payment' page to 'reporting and reconciling payments'
- changes the focus of the reporting page to reconciliation
- fixes any broken links caused by the changes
- adds metadata content to reconciliation guidance
- removes the metadata page

### CODE REVIEW

File: source/reporting/index.html.md.erb [hunk 1]
```diff
@@ -1,43 +1,77 @@
...
+You can change what is shown on your bank statement for the payment in the Worldpay settings.
 
-    If you have more than 5000 transactions, you must apply a filter before the CSV file download link appears.
+### How Stripe payouts work
 
-The CSV file contains all transaction data, which is much more comprehensive than the information on the __Transactions__ page.
+You will receive one payment into your bank account for each Stripe service you have in Pay. 
 
-You can process CSV files using spreadsheet software and edit the contents you do not need.
+You can choose to have multiple types of transaction going through one service (e.g. licensing and council tax, or parking fines and parking permits), or you can choose to have separate services for each type of transaction. You can set the description for each transaction type so it’s clear what has been paid for.
```
Comment (stephencdaly) [inline source/reporting/index.html.md.erb:36]: Do we need to elaborate on what we mean by "You can set the description"?

File: source/reporting/index.html.md.erb [hunk 2]
```diff
@@ -1,43 +1,77 @@
...
+- the user’s card details
+
+
+When you select a payment, you can see the history of the payment. For example, you can see when your service created the payment and when the user paid.
+
+
+You can also download a CSV spreadsheet of all your selected payments to help with [reconciliation](#check-your-payments-are-correct-39-reconciliation-39). You can download one report for all your services or one report for each separate service.
+
+
+You can also see payments made to your bank account by following the link from **My services** but only once you have [gone live](/switching_to_live) or set up a Stripe test account.
```
Comment (stephencdaly) [inline source/reporting/index.html.md.erb:72]: Does this need to make clear this is only available for Stripe live accounts?

File: source/reporting/index.html.md.erb [hunk 3]
```diff
@@ -284,4 +317,142 @@ Example search response:
...
+
+The GOV.UK Pay API can give you payment settlement data in the `settled_date` attribute in `YYYY-MM-DD` format. You can also find payments based on when they were settled.
+
+To get settlement information about a single payment, [get information about that payment](#get-information-about-a-single-payment) through our API.
+
+To get settlement information about multiple payments, [generate a list of payments](#generate-a-list-of-payments-matching-search-criteria) through our API. Use the `from_settled_date` and `to_settled_date` query parameters to only include payments that were settled within a specific date range.
+
+Read more about the `settlement_summary` object in our API reference pages for [getting information about a single payment](/api_reference/single_payment_reference) and [searching for payments](/api_reference/search_payments_reference).
+
+##### Generate a list of payments that match your search criteria
```
Comment (stephencdaly) [inline source/reporting/index.html.md.erb:444]: Is "Generate" the right word here? Why not just "get"?

Comment (stephencdaly) [reply source/reporting/index.html.md.erb:444]: Could this title be something like "Get a list of payments by settled date" or something similar? I thought this was repeating what is higher up on this page, but then I saw this is talking in particular about filtering by settled date.

File: source/reporting/index.html.md.erb [hunk 4]
```diff
@@ -69,7 +69,7 @@ When you select a payment, you can see the history of the payment. For example,
 You can also download a CSV spreadsheet of all your selected payments to help with [reconciliation](#check-your-payments-are-correct-39-reconciliation-39). You can download one report for all your services or one report for each separate service.
 
 
-You can also see payments made to your bank account by following the link from **My services** but only once you have [gone live](/switching_to_live) or set up a Stripe test account.
+You can also see payments made to your bank account by following the link from **My services**. This is only available for live Stripe accounts.
```
Comment (stephencdaly) [inline source/reporting/index.html.md.erb:72]: Also for test Stripe accounts, sorry, which we said previously. Just thought it needed clarifying that it wasn't for *all* live accounts but only test Stripe accounts.

File: source/reporting/index.html.md.erb [hunk 5]
```diff
@@ -25,15 +25,15 @@ You will receive one payment into your bank account for each Worldpay merchant c
 
 If you have more than one service in Pay, you can choose to use the same merchant code for each service or a different merchant code for each one. 
 
-If you have multiple services on one merchant code you can use our reporting API to find all the transactions that are on that merchant code, or you can use the All live transactions report to download a CSV with this information from the admin tool. (The merchant code itself is not shown in the API or the CSV, but you can use the description line to find the relevant transactions).
+If you have multiple services on one merchant code, you can use our reporting API to find all the transactions that are on that merchant code. Select **View transactions for all your service** in the admin tool to download a CSV with this information. The merchant code itself is not shown in the API or the CSV, but you can use the description line to find the relevant transactions.
```
[the remark this answers is not in the document]
Comment (NathanD-GDS) [reply source/reporting/index.html.md.erb:28]: Agreed

File: source/reporting/index.html.md.erb [hunk 6]
```diff
@@ -25,7 +25,7 @@ You will receive one payment into your bank account for each Worldpay merchant c
 
 If you have more than one service in Pay, you can choose to use the same merchant code for each service or a different merchant code for each one. 
 
-If you have multiple services on one merchant code, you can use our reporting API to find all the transactions that are on that merchant code. Select **View transactions for all your service** in the admin tool to download a CSV with this information. The merchant code itself is not shown in the API or the CSV, but you can use the description line to find the relevant transactions.
+If you have multiple services on one merchant code, you can use our reporting API to find all the transactions that are on that merchant code. Alternatively, select **View transactions for all your service** in the admin tool to download a CSV with this information. The merchant code itself is not shown in the API or the CSV, but you can use the description line to find the relevant transactions.
```
Comment (stephencdaly) [inline source/reporting/index.html.md.erb:28]:
```suggestion
If you have multiple services on one merchant code, you can use our reporting API to find all the transactions that are on that merchant code. Alternatively, select **View transactions for all your services** in the admin tool to download a CSV with this information. The merchant code itself is not shown in the API or the CSV, but you can use the description line to find the relevant transactions.
```

**Review by stephencdaly — APPROVED** [review]
Looks good to me!

**Pull request merged**